apiClient.go

```diff
@@ -55,35 +63,28 @@ type LWAPIRes interface {
 	HadError() bool
 }
 
-// New takes a *viper.Viper, and gives you a *Client. If there's an error, it is returned.
+// New takes a *LWAPIConfig, and gives you a *Client. If there's an error, it is returned.
 // When using this package, this should be the first function you call. Below is an example
 // that demonstrates creating the *viper.Viper and passing it to New.
```

Comment:
might as well remove this old `*viper.Viper` reference in the comment as well

Reply:
ah, good catch.